doc/install.txt

```diff
@@ -4,891 +4,15 @@
 Installing Theano
 =================
 
+On the following pages you will find different installation instructions depending
+on your needs. Each instruction will take care of the :ref:`requirements`.
```

Comment:
Let's put the reference to the requirements *after* the installation instruction, to make sure people don't go straight there and try to install everything. Also, add something like "if you are curious about them" or similar.